bufferGenerators.js

```diff
@@ -14,7 +14,7 @@ class BufferGenerators {
 
   static positionBuffer(position) {
     var x = position.x
-    var y = position.y
+    var y = position.y ? position.y : 16
```

Comment:
right now I'm only really concerned about the players ground position so I'm allowing this buffer to be used without a y value